### ISSUE TITLE
Pure Data won't make any sound until UI is displayed via a X-Window on a PC

### ISSUE THREAD

**Issue by MrDham** [zynthian/zynthian-issue-tracking#50]
**Describe the bug**
Pure Date presets don't make any sound if Zynthian is not connected to a PC and if UI is not displayed on this PC (by using "ssh -Y root@zynthian.local")

**To Reproduce**
Case 1:
Connect Zynthian to an audio amplifier
Create a new layer with PD > generative/generative_relaxing

Result: No sound can be heard

Case 2
Connect Zynthian to audio amplifier
Connect a PC via ethernet and log to a X-Window using  "ssh -Y root@zynthian.local"
Create a new layer with PD > generative/generative_relaxing
Result : The UI is dispayed on the PC and some automatically generated sound can be heard

**Expected behavior**
We should hear an automatically generated sound in both case 1 and 2. 
But we have some sound in case 2 only. 

**Screenshots**
N/A

**Setup:**
 - SD-image
Aruk RC3
 - Hardware Config
Raspberry Pi 3 Model B Plus Rev 1.3
Soundcard: HifiBerry DAC+
Display: PiScreen 3.5 (v2)
Wiring: MCP23017_EXTRA
 - Software Version
zyncoder: master (80862b6)
zynthian-ui: master (3688c73)
zynthian-sys: master (26f86d2)
zynthian-data: master (676c32a)
zynthian-webconf: master (0094c99)

**Additional context**
PC conf for X-Window: Xming + Putty on a windows PC (for case 2)

**Comment by jofemodo** [comment]
Sorry. I can't reproduce the problem. For me it's working like a charm ...
Could somebody else check it?

**Comment by MrDham** [comment]
Hi. 
I made some further investigations.  
I tested it with Aruk RC2 (without any update) and it works well. 
The problem seem to be with Aruk RC3.

**Comment by jofemodo** [comment]
You are right. I'm trying to fix it but i've no idea where the problem is ... or better, i can't find any difference between RC-2 and RC-3 regarding puredata ;-)

**Comment by MrDham** [comment]
I just made some further investigation. 
In RC2, I can get same symptom with the following scenario:  
  Start Zynthian with no layer. 
  Open a X window using "ssh -Y root@zynthian.local"
  Enter password and then close the session. 
  Then create a new layer with  PD > generative/generative_relaxing. 

       No sound can be heard.  

  Reboot
       
      Some relaxing sound can be heard

Maybe search the bug on X window side (a "phantom" x window session in RC3, ...  ?)

I hope it will help :-)

**Comment by MrDham** [comment]
Mistakenly closed when posting my comment.
